dusty/commands/script.py

```diff
@@ -10,14 +10,14 @@ def script_info_for_app(app_name):
     app_specs = get_specs()['apps'].get(app_name)
     if not app_specs:
         raise KeyError('No app found named {} in specs'.format(app_name))
-    if not app_specs.get('scripts'):
+    if len(app_specs['scripts']) == 0:
```

Comment:
we default scripts to a list now.  But you might be asking why I am not just checking that it is an empty list.  That is a good point